utils/errors/errors.go

```diff
@@ -77,9 +77,6 @@ func (c customError) Cause() error {
 }
 
 func (c customError) GRPCStatus() *grpcstatus.Status {
-	if c.status == nil {
```

Comment:
Since the default status will be given in the `WrapWithSkipAndStatus` function in this change, c.Status will always not be nil and thus can be return without checking.

Reply:
this nil check was mainly for backward-compatibility, if old version or go-utils generates customError without status, then here we still return nil. I'd suggest we keep this for a longer time unless your team faced any critical issues regarding this